clang/www/make_cxx_dr_status

```diff
@@ -159,21 +165,37 @@ def availability(issue):
   if status == 'unknown':
     avail = 'Unknown'
     avail_style = ' class="unknown"'
-  elif re.match('^[0-9]+\.?[0-9]*', status):
-    avail = 'Clang %s' % status
-    if float(status) > latest_release:
-      avail_style = ' class="unreleased"'
-    else:
-      avail_style = ' class="full"'
+  elif re.match(r'^[0-9]+\.?[0-9]*', status):
+    if not proposed_resolution:
+      avail = 'Clang %s' % status
+      if float(status) > latest_release:
+        avail_style = ' class="unreleased"'
+      else:
+        avail_style = ' class="full"'
+    else: 
+      avail = 'Not Resolved*'
+      avail_style = f' title="Clang {status} implements {proposed_resolution} resolution"'
   elif status == 'yes':
-    avail = 'Yes'
-    avail_style = ' class="full"'
+    if not proposed_resolution:
+      avail = 'Yes'
+      avail_style = ' class="full"'
+    else:
+      avail = 'Not Resolved*'
+      avail_style = f' title="Clang implements {proposed_resolution} resolution"'
   elif status == 'partial':
-    avail = 'Partial'
-    avail_style = ' class="partial"'
+    if not proposed_resolution:
+      avail = 'Partial'
+      avail_style = ' class="partial"'
+    else:
+      avail = 'Not Resolved*'
+      avail_style = f' title="Clang partially implements {proposed_resolution} resolution"'
```

Comment:
Does this case happen?

Reply:
I'm fine keeping it on the assumption it might be used in the future, but I'm also fine stripping it out on the assumption that it's easy enough to add support for once we have an actual need for it.